src/odemis/acq/stitching/test/tiledacq_test.py

```diff
@@ -490,11 +491,14 @@ def test_estimate_time(self):
                                               zlevels=[1, 2, 3])
         tiled_acq_task._nx = 7
         tiled_acq_task._ny = 7
-        # observed time duration during stage movement between tiles
-        min_stage_time = tiled_acq_task._nx * tiled_acq_task._ny * 0.3
-        # observed time duration during tile acquisition
-        min_ties_acq = tiled_acq_task._nx * tiled_acq_task._ny * len(tiled_acq_task._zlevels)
-        min_total_time = min_stage_time + min_ties_acq
+        total_tiles = tiled_acq_task._nx * tiled_acq_task._ny
+        # Observed time duration during stage movement between tiles
+        min_stage_time = total_tiles * 0.3
+        # Calculated time duration during tile acquisition
+        zlevels_dict = {s: tiled_acq_task._zlevels for s in tiled_acq_task._streams
+                        if isinstance(s, (FluoStream))}
+        acq_time = acqmng.estimateZStackAcquisitionTime(tiled_acq_task._streams, zlevels_dict) * total_tiles
+        min_total_time = min_stage_time + acq_time
         # overhead of 30% on min_total_time is used to account for stream settings, like, exposure time etc.
         self.assertTrue(min_total_time <= tiled_acq_task.estimateTime() <= 1.30 * min_total_time)
```

Comment:
while you are fixing this test case, can you also replace this `assertTrue`. `assertTrue` should be used when checking if a certain boolean is True, for this case `assertLess` and `assertGreater` work better (please double check if I didn't make a mistake in this suggestion). This will also make sure the error message is more informative
```suggestion
        estimated_time = tiled_acq_task.estimateTime()
        self.assertLessEqual(min_total_time, estimated_time )
        self.assertLessEqual(estimated_time, 1.30 * min_total_time)
```